index.js

```diff
@@ -510,13 +510,13 @@ exports.generateAddress = function (from, nonce) {
 }
 
 /**
- * Returns true if the supplied address belongs to a precompiled account
+ * Returns true if the supplied address belongs to a precompiled account (Byzantium)
  * @param {Buffer|String} address
  * @return {Boolean}
  */
 exports.isPrecompiled = function (address) {
   const a = exports.unpad(address)
```

Comment:
Add `isValidAddress`?

Reply:
Hmm, normally your are right. I wouldn't want this here, since the ``IsPrecompiled`` method is removed anyhow (citing @axic) on next major release. Adding ``isValidAddress`` would change the behavior here and would cause extra/unnecessary problems for people already have this in use.